rslib/src/scheduler/states/review.rs

```diff
@@ -63,6 +63,17 @@ impl ReviewState {
     pub(crate) fn next_states(self, ctx: &StateContext) -> SchedulingStates {
         let (hard_interval, good_interval, easy_interval) = self.passing_review_intervals(ctx);
```

Comment:
These three intervals have been fuzzed. I recommend disabling the fuzz when load balancer is enabled.

Reply:
oh good catch, I did that in a previous iteration of this to this but it seems I forgot to do it this time